Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx

```diff
@@ -107,7 +116,55 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
 
       // use the altro decoder to decode the raw data, and extract the RCU trailer
       o2::emcal::AltroDecoder decoder(rawreader);
-      decoder.decode();
+      //check the words of the payload exception in altrodecoder
+      try {
+        decoder.decode();
+      } catch (AltroDecoderError& e) {
+        std::stringstream errormessage;
+        using AltroErrType = o2::emcal::AltroDecoderError::ErrorType_t;
+        int errornum = -1;
+        switch (e.getErrorType()) {
+          case AltroErrType::RCU_TRAILER_ERROR:
+            errornum = 0;
```

Comment:
Wherever possible I would like to avoid hard coding of numeric constants in favour of symbolic constants. Numeric constants might change their meaning when code evolves, while the meaning of a symbolic constant stays well defined. In case we expand the error codes there should be a common header file with an enum for error codes. For now porting the AltroErrorType would be the more appropriate solution.